install_config/persistent_storage/storage_classes.adoc

```diff
@@ -12,22 +12,25 @@
 toc::[]
 
 == Overview
-The *StorageClass* resource object is used to describe and classify storage that can be requested, as well as 
-provide a means for passing in parameters for *dynamically provisioned storage* on-demand.  StorageClasses can also serve
-as a management mechanism for controlling different levels of storage and access to the storage.  Cluster Administrators (cluster-admin)
-or Storage Administrators (storage-admin) need to define and create the StorageClasses that can then be requested by users without the users needing to have
-any intimate knowledge about the underlying storage volume sources.
-
-The Kubernetes
+The *StorageClass* resource object describes and classifies storage that can be
+requested, as well as provide a means for passing parameters for
+*dynamically provisioned storage* on demand. StorageClasses can also serve as
```

Comment:
I marked up he instance of it. Wasn't sure if the rest of the mentions needed markup.